int16 inflatable_state
